### PR TITLE
Fix *.pdb files missing in meson introspect --installed output

### PR DESCRIPTION
**Pull request by xhaakon**
On Windows, make sure the introspect command lists all Program database
(PDB) files containing debugging information that Meson will install.

### CODE REVIEW

File: mesonbuild/build.py [hunk 1]
```diff
@@ -1490,6 +1493,11 @@ def __init__(self, name, subdir, subproject, for_machine: MachineChoice, sources
                 else:
                     self.import_filename = self.gcc_import_filename
 
+        if m.is_windows() and ('cs' in self.compilers
+                               or self.get_using_rustc()
+                               or self.get_using_msvc()):
```
Comment (ignatenkobrain) [inline mesonbuild/build.py]: __[Flake8]__ [W503] line break before binary operator ([link](https://sider.review/gh/repos/19784232/pulls/5907#issue-4555850))

<sub>posted by Sider</sub>

File: mesonbuild/build.py [hunk 2]
```diff
@@ -1490,6 +1493,11 @@ def __init__(self, name, subdir, subproject, for_machine: MachineChoice, sources
                 else:
                     self.import_filename = self.gcc_import_filename
 
+        if m.is_windows() and ('cs' in self.compilers
+                               or self.get_using_rustc()
```
Comment (ignatenkobrain) [inline mesonbuild/build.py]: __[Flake8]__ [W503] line break before binary operator ([link](https://sider.review/gh/repos/19784232/pulls/5907#issue-4555849))

<sub>posted by Sider</sub>

**Comment by jpakkane** [comment]
There should be a test checking that the pdb file is correctly written in the introspect output.

**Comment by xhaakon** [comment]
New test case added.

**Pull request merged**